api/v1alpha1/wasm_types.go

```diff
@@ -79,7 +79,7 @@ type WasmCodeSource struct {
 
 	// SHA256 checksum that will be used to verify the wasm code.
 	// +optional
-	SHA256 *string `json:"sha256,omitempty"`
+	// SHA256 *string `json:"sha256,omitempty"`
```

Comment:
should remove it instead of comment.

Reply:
I leave this because it's likely to be used when implementing wasm extension.  

Of course, I'll remove it if I find out it's not necessary when implementing it.